### PR TITLE
Add docs for MSI install

### PR DESCRIPTION
**Pull request by kilfoyle**
Adds docs for installing Elastic Agent via the newly supported MSI packages, which will GA in 8.14.

Thanks for the nice draft, Amit! Please see [PREVIEW PAGE](https://ingest-docs_bk_1039.docs-preview.app.elstc.co/guide/en/fleet/master/install-agent-msi.html)

Closes: #1036

### CODE REVIEW

File: docs/en/ingest-management/elastic-agent/install-agent-msi.asciidoc [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+Upgrades are not supported and are prevented by the MSI itself. Instead, all of your {agent} upgrades can be managed in {fleet}.
+
+[discrete]
+== Installing in a custom location
+
+Starting in version 8.13, it's also possible to override the default installation folder by running the MSI from the command line, as shown:
+
+[source,shell]
+----
+msiexec /i "<full path to msi file>" INSTALLDIR="<path of custom folder>"
```
Comment (amitkanfer) [inline docs/en/ingest-management/elastic-agent/install-agent-msi.asciidoc:59]: This is not clear enough. I'd change to have the example from line [17](https://github.com/elastic/ingest-docs/pull/1039/files#diff-0d2c2993fe4d3ef63fa65aeba2a4656e32359858b31f43853473a2591e6d94c7R17) with the addition of `INSTALLDIR="<path of custom folder>"` at the end

**Review by amitkanfer — CHANGES_REQUESTED** [review]

File: docs/en/ingest-management/elastic-agent/install-agent-msi.asciidoc (hunk 1 above)
Comment (kilfoyle) [reply docs/en/ingest-management/elastic-agent/install-agent-msi.asciidoc]: Good idea! Fixed in [135fd6a](https://github.com/elastic/ingest-docs/pull/1039/commits/135fd6a5d3a42d21298a9ce4e28188de2a1b418b) and changed to:

`elastic-agent-<VERSION>-windows-x86_64.msi INSTALLARGS="--url=<URL> --enrollment-token=<TOKEN>" INSTALLDIR="<path of custom folder>"`

**Review by amitkanfer — APPROVED** [review]

**Comment by amitkanfer** [comment]
🚀

**Review by karenzone — APPROVED** [review]
LGTM  🚀

**Pull request merged**